src/components/MainScheduler/index.jsx

```diff
@@ -0,0 +1,220 @@
+import * as React from 'react';
+import Paper from '@mui/material/Paper';
+import ToggleButton from '@mui/material/ToggleButton';
+import ToggleButtonGroup from '@mui/material/ToggleButtonGroup';
+import PropTypes from 'prop-types';
+import { ViewState } from '@devexpress/dx-react-scheduler';
+import {
+    Scheduler,
+    DayView,
+    WeekView,
+    MonthView,
+    Toolbar,
+    DateNavigator,
+    Appointments,
+    TodayButton,
+} from '@devexpress/dx-react-scheduler-material-ui';
+
+const appointments = [

```

Comment:
This is only mock data, right?

I think that @jferdi24  (Diego Lozano) and @JimmyAnaya needs to prepare something in the backend for this

Reply:
Yes, we need to talk about it. It must work with the interviews flow